src/Common/Admin/Entities/Div.php

```diff
@@ -21,12 +22,17 @@ class Div extends Container {
 	/**
 	 * Div constructor.
 	 *
-	 * @param ?Element_Classes $classes The classes for the div.
+	 * @param ?Classes    $classes    The classes for the div.
+	 * @param ?Attributes $attributes The attributes for the div.
 	 */
-	public function __construct( ?Element_Classes $classes = null ) {
+	public function __construct( ?Classes $classes = null, ?Attributes $attributes = null ) {
```

Comment:
That's a good question. It crossed my mind, but at this stage I figured that anyone using this will be controlling what goes into it, so hopefully they know not to put classes in both places.